lib/puppetserver/ca/action/list.rb

```diff
@@ -47,6 +47,9 @@ def self.parser(parsed = {})
             opts.on('--all', 'List all certificates') do |a|
               parsed['all'] = true
             end
+            opts.on('--format FORMAT', "Valide formats are: 'text'(default), 'json") do |f|
```

Comment:
since I can't make a suggestion on Justin's suggestion..
```suggestion
            opts.on('--format FORMAT', "Valid formats are: 'text' (default), 'json'") do |f|
```

Reply:
lol